pandas/tests/generic/test_to_xarray.py

```diff
@@ -37,7 +37,7 @@ def test_to_xarray_index_types(self, index_flat, df, using_infer_string, request
             pytest.skip("Test doesn't make sense for empty index")
         import xarray
 
-        if Version(xarray.__version__) >= Version("2024.5"):
+        if Version(xarray.__version__) == Version("2024.5"):
```

Comment:
Typically we just remove these xfails if a latest optional dependency fixes a test.